Cross Region VM Restore Points.md

```diff
@@ -0,0 +1,396 @@
+               }
+            }
+         ]
+      },
+      "diagnosticsProfile": {
+         "bootDiagnostics": {
+            "enabled": true,
+            "storageUri": " http://storageaccount.blob.core.windows.net/"
+         }
+      },
+   },
+   "provisioningState": "Succeeded | Failed | Creating | Deleting",
+   "provisioningDetails": {
+      "creationTime": "Creation Time of Restore point in UTC",
+   },
+   "instanceView": {
+      "statuses": [{
+          "code": "ReplicationState/succeeded",
+          "level": "Info",
+          "displayStatus": "Replication succeeded",
+          "message": "<will be populated in error scenarios>",
+          "time": "2019-10-14T21:29:47.477089+00:00"
+      }],
+      "diskRestorePoints": [{
+          "id": "<diskRestorePoint Arm Id>",
+  “replicationStatus” : {
+"status": {
+          	"code": "ReplicationState/succeeded",
+          	"level": "Info",
+          	"displayStatus": "Replication succeeded",
+          	"message": "<will be populated in error scenarios>",
+          	"time": "2019-10-14T21:29:47.477089+00:00"
+          	}
+  “completionPercent” : <completion percentage of the replication>
+}
+      }],
+    }  
+}
+```
+
+## Known issues and limitations
+* Cross-region copy functionality is only supported via REST APIs. ARM Template deployment, CLI, SDK, PS will be added later. 
+* Copy of copy does not work - You cannot copy a restore point that is already copied from another region. For ex. if you have copied a RP1 from East US to West US as RRP1. You cannot copy RRP1 from West US to another region (or back to East US). 
+* A single Restore Point in the source region can only be copied once to a target region. Meaning you cannot have multiple copies of the same restore point in a single target region.
+* Copy of the Restore point of CMK encrypted VM and disks will be encrypted using PMK in the target region. Customer needs to use CMK when restoring the disks and VM from the restore point.
+* Target Restore Point only shows the creation time when the target Restore Point was created. To get the creation time of the original restore point, you need to call GET on the original restore point. 
+* Currently, the replication progress in only updated once every 10mins. Hence for disks that have low churn, there can be scenarios where only the initial (0) and the terminal replication progress (100) can be seen.
+* Maximum copy time that is supported is 2 weeks. If there is huge amount data to be copied to the target region, depending on the bandwidth available between the regions, the copy time could be couple of days. If the copy time exceeds 2 weeks, the copy operation will be terminated. 
+* There are no error details provided when a Disk Restore Point copy fails
+* When a disk restore point copy fails, its completion percent is shown as 0, not the intermediate value where it stopped copying.
```

Comment:
We do not show any completion percent at all when disk restore point copy fails. It would be great if you could confirm with Shrutika once on this behavior.

Reply:
Confirmed with Shrutika. We do not show percentage for failure. I have made the change.